packages/services/src/view-service/unbonding-tokens-by-address-index/index.test.ts

```diff
@@ -71,13 +83,25 @@ const mockBalancesResponse3 = new BalancesResponse({
       value: {
         amount: { hi: 0n, lo: 3n },
         metadata: {
-          display: 'unbonding_start_at_200_penumbravalid1abc123',
+          display:
+            'unbonding_start_at_200_penumbravalid1ltltrqe7f2c0q8mqlsx3u74s0r6nm968f325njz4h8zzqh0gsgfq2g5d3m',
         },
       },
     },
   },
 });
 
+const mockBalancesResponse4 = new BalancesResponse({
+  balanceView: {
+    valueView: {
+      case: 'unknownAssetId',
+      value: {
+        amount: { hi: 0n, lo: 3n },
+      },
+    },
+  },
+});
```

Comment:
added to make sure that having balances with unknownAssetId does not break anything